.github/workflows/tests.yml

```diff
@@ -30,6 +30,7 @@ jobs:
       PUMA_TEST_DEBUG: true
       TESTOPTS: -v
       PUMA_NO_RUBOCOP: true
+      TERM: BugMinitest
```

Comment:
This just hit my radar yesterday and I didn't change anything to do with how pride works in this release... did GHA recently remove the TERM variable?? What's going on? This is new, yeah?

(funny fix btw... @flavorjones disabled pride in mechanize)

Reply:
For Runner info, I created a repo https://github.com/MSP-Greg/actions-image-testing/actions, see the 'show ENV' step.

At present, it seems none of the runners define TERM.  the older line shown above will accept `nil`, `match?` will not.  No biggy.  Was this a RuboCop change? Maybe their docs should mention the `nil`` issue...  Haven't looked.